doc/conf.py

```diff
@@ -105,11 +105,11 @@
     "examples_dirs": [
         "../examples/gallery",
```

Comment:
Please also sort lines 106-110, lines 115-132, and line 112.

Reply:
See commit https://github.com/GenericMappingTools/pygmt/commit/e2995083de6e8bdbd0f1b3056f52263547965f07.